docs/mdbook/src/android/getting-started-guide.md

```diff
@@ -0,0 +1,118 @@
+
+    ```xml
+        ...
+        <com.mapbox.mapboxsdk.maps.MapView
+            android:id="@+id/mapView"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            />
+        ...
+    ```
+
+5. Initialize the `MapView` in your `MainActivity` file by following the example below:
+
+    ```kotlin
+
+        ...
+        import androidx.appcompat.app.AppCompatActivity
+        import android.os.Bundle
+        import android.view.LayoutInflater
+        import com.mapbox.mapboxsdk.Mapbox
+        import com.mapbox.mapboxsdk.camera.CameraPosition
+        import com.mapbox.mapboxsdk.geometry.LatLng
+        import com.mapbox.mapboxsdk.maps.MapView
+
+        class MainActivity : AppCompatActivity() {
+
+            // Declare a variable for MapView
+            private var mapView: MapView? = null
+
+            override fun onCreate(savedInstanceState: Bundle?) {
+                super.onCreate(savedInstanceState)
+                
+                // Init MapLibre
+                Mapbox.getInstance(this)
+
+                // Init layout view
+                val inflater = LayoutInflater.from(this)
+                val rootView = inflater.inflate(R.layout.activity_main, null)
+                setContentView(rootView)
+
+                // Init the MapView
+                mapView = rootView.findViewById(R.id.mapView)
+                mapView.getMapAsync { map ->
+                    map.setStyle("https://demotiles.maplibre.org/style.json")
+                    map.cameraPosition = CameraPosition.Builder().target(LatLng(0.0,0.0)).zoom(1.0).build()
+                }
+            }
+
+            override fun onStart() {
+                super.onStart()
+                mapView?.onStart()
+            }
+
+            override fun onResume() {
+                super.onResume()
+                mapView?.onResume()
+            }
+
+            override fun onPause() {
+                super.onPause()
+                mapView?.onPause()
+            }
+
+            override fun onStop() {
+                super.onStop()
+                mapView?.onStop()
+            }
+
+            override fun onLowMemory() {
+                super.onLowMemory()
+                mapView?.onLowMemory()
+            }
+
+            override fun onDestroy() {
+                super.onDestroy()
+                mapView?.onDestroy()
+            }
+
+            override fun onSaveInstanceState(outState: Bundle) {
+                super.onSaveInstanceState(outState)
+                mapView?.onSaveInstanceState(outState)
+            }
+        }
+        
+    ```
+
+6. Build and run the app. If you run the app successfully, a map will be displayed as seen in the screenshot below.
+<div style="text-align: center;">
+<img src="/assets/MapLibre_Native_Android_Demo_Screenshot.png" alt="Screenshot with the map in demotile style" width="50%" height="50%">
```

Comment:
Please don't include screenshots in the repo. This will blow up the size of the repo over time. You can host them on GitHub. Drag and drop an image in the GitHub editor (anywhere) to get an URL.

Reply:
I removed the screenshot from the repos and hosted it directly via Github editor in these commits 7af2f822ec73c936f46ba079fff0d8da48680e78 and a938d5ceb4b9395aa1c71afb10c6f592f00345af.